### PR TITLE
add autocomplete attributes

### PR DESCRIPTION
**Pull request by xi**
see https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute

### CODE REVIEW

**Comment by davidsomnus** [comment]
Tested this with django-allauth (0.41.0) and it works nicely.

File: allauth/account/forms.py [hunk 1]
```diff
@@ -127,8 +134,9 @@ def __init__(self, *args, **kwargs):
             assert app_settings.AUTHENTICATION_METHOD \
                 == AuthenticationMethod.USERNAME_EMAIL
             login_widget = forms.TextInput(attrs={'placeholder':
-                                                  _('Username or e-mail'),
-                                                  'autofocus': 'autofocus'})
+                                                  _('Username e-mail'),
```
Comment (pennersr) [inline allauth/account/forms.py:137]: Why did you remove "or" here?

File: allauth/account/forms.py [hunk 2]
```diff
@@ -127,8 +134,9 @@ def __init__(self, *args, **kwargs):
...
+                                                  'autofocus': 'autofocus',
+                                                  'autocomplete': 'username'})
```
Comment (pennersr) [inline allauth/account/forms.py:139]: Wouldn't autocomplete=login make more sense, given that it is either the username or the email?

File: allauth/account/forms.py [hunk 3]
```diff
@@ -111,14 +116,16 @@ def __init__(self, *args, **kwargs):
             login_widget = forms.TextInput(attrs={'type': 'email',
                                                   'placeholder':
                                                   _('E-mail address'),
-                                                  'autofocus': 'autofocus'})
+                                                  'autofocus': 'autofocus',
```
Comment (pennersr) [inline allauth/account/forms.py:119]: autofocus was dropped, so it should be removed from this PR as well.

**Review by pennersr — CHANGES_REQUESTED** [review]

**Comment by xi** [comment]
This PR is 3 years old and I am no longer using django-allauth. I don't have time to work on this so it is up for grabs.

File: allauth/account/forms.py (hunk 1 above)
Comment (xi) [reply allauth/account/forms.py:137]: Probably unintentional. Feel free to restore the original string.

**Comment by pennersr** [comment]
Closed via 239a7235
